test/ci/env/global.env

```diff
@@ -11,8 +11,9 @@ BASEX_VERSION=10.1
 # Do not perform Maven package by default
 DO_MAVEN_PACKAGE=
 
-# Log4j is for XML Calabash 1.3.2 or later
-LOG4J_VERSION=2.17.1
+# SLF4J is for XML Calabash 1.3.2 or later.
+# XML Calabash has slf4j-api-1.7.36.jar in its lib directory.
```

Comment:
> Should we be upgrading to XML Calabash version 1.5.4-100 anyway (could be a separate PR if you like)?

> Upgrading to XML Calabash version 1.5.4-100 might require an XSLT change like [this](https://github.com/xspec/xspec/pull/1717/files#diff-2a493ef5a2355cd4bd3ebfb2155c363ffcf77c18c9eb12840ea75a7203d429a4).

Opened ndw/xmlcalabash1#343 to seek clarification on what happened. After getting a response, let's decide what to do with 1.5.4-100.

Reply:
> `saxon-10.env` lists XML Calabash version 1.3.2-100, which has `slf4j-api-1.7.30.jar` in its lib directory.

The main purpose of this comment is to explain why a preference exists for 1.7 over 2.0. I updated the comment in 3e0037d6325c211c848f1ef5edc967f8af1bf455.
Given that 1.7.36 is the latest (and seems like the last) release of 1.7, it would be optimal to select 1.7.36.

Reply:
Perfect, thanks.